test/deal-folder/post-document-in-deal-folder.api-test.ts

```diff
@@ -26,10 +27,14 @@ describe('postDocumentInDealFolder', () => {
     getItemIdPath,
     getItemIdResponse,
     updateFileInfoPath,
-    updateFileInfoRequest,
   } = new UploadFileInDealFolderGenerator(valueGenerator).generate({ numberToGenerate: 1 });
   const [{ fileName, fileLocationPath }] = uploadFileInDealFolderRequest;
   const { siteId, dealId } = uploadFileInDealFolderParams;
+  const updateFileInfoRequest = {
```

Comment:
Happy to do it this way -- in other places, we do have the ability to pass in env vars into the generator (just for your knowledge, can resolve once read!)

Reply:
Changed to the approach you describe